src/controller/cve.controller/index.js

```diff
@@ -346,7 +322,7 @@ router.post('/cve/:id/cna',
     }
   }
   #swagger.responses[200] = {
-    description: 'The requested CVE record is returned.',
+    description: 'The requested CVE record is returned',
     content: {
       "application/json": {
         schema: { $ref: '/schemas/cve/create-cve-record-response.json' }
```

Comment:
Response body for PUT /cve/:id/cna will be almost the same as the POST - need to change 'created' to 'updated'